### DISCUSSION TITLE
Lack of WindiCSS desktop first breakpoint variants like "<md"

### DISCUSSION THREAD

**Discussion by nvhoang55** [unocss/unocss#997] [Q&A] [answered]
So I can't use any of the Windi CSS desktop first breakpoint variants, and I have to find a workaround. For example, when I need to center the logo only on < md screen and do nothing else, I can do it like this: "**`<md:(flex justify-center w-full)`**". Of course, I can get the job done with "**`flex justify-center w-full md:(justify-start w-max)`**".

But, I'm keen on the first option.

Does anyone know how to add those variants?

**Comment by wheatjs** [comment, marked as answer]
In unocss we don't use < or >, instead `lt-md:(flex justify-center w-full)` and for `>` use `gt-`

**Reply by nvhoang55** [reply]
Thank you a lot <3
Btw, do you know where those conventions are mentioned?

**Reply by wheatjs** [reply]
afaik I think this is the only place that it is mentioned currently https://github.com/unocss/unocss/issues/181#issuecomment-982443908

**Reply by nvhoang55** [reply]
Gonna give it a look, thanks

**Reply by wheatjs** [reply]
Might be something we can document a bit better, I'll make a note of it and if I have time see if I can improve the current situation a little
